Content.Server/_NF/Security/Components/ContrabandGunComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Server._NF.Security.Components;
+
+/// <summary>
+/// This is used for the contraband gun, which detect contraband status of any object it appraises.
```

Comment:
We can do this later after we refactor the contraband comp.

Reply:
the contraband comp designates a currency type iirc, so that you can have different 'kinds' of contraband. its just currently unused, but can easily be expanded so that different items are considered contraband to different 'factions'